apps/dataflow/src/scraper/fms/s3_utils.py

```
@@ -0,0 +1,248 @@
+def get_generic_file_from_s3(s3_object_name):
+    """
+    Get generic data from S3 bucket
+
+    Args:
+        s3_object_name (str): The S3 object name/path
+        return_data (bool): If True, return the data as Python object
+                           If False, return the raw response object
+
+    Returns:
+        response object
+    """
```

Ruff (0.14.14)

[warning] Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)